presto-main/src/main/java/com/facebook/presto/server/ServerConfig.java

```diff
@@ -38,6 +40,30 @@ public ServerConfig setCoordinator(boolean coordinator)
         return this;
     }
 
+    public boolean isEnabledUIonSecondaryPort()
+    {
+        return enabledUIonSecondaryPort;
+    }
+
+    @Config("http-server.ui.secondary.port.enabled")
```

Comment:
+1 to @dabaitu's comment. A user shouldn't have to set a port and also set a flag that they're setting a port.